client/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+    <!-- Component styles -->
+    <link rel="stylesheet" href="../components/button/button.css">
```

Comment:
oh cool idea with such splitting.
I think we could also consolidate it in one stylesheet with css @import()!

Reply:
Hell, yes! Gonna create the consolidated stylesheet.

Great suggestion, thank you!